### PR TITLE
Rollups API spec

### PR DESCRIPTION
**Pull request by djmadeira**
### Description
Add specs for rollup APIs

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +0,0 @@
-{
-  "name": "opensearch-api-specification",
-  "lockfileVersion": 3,
-  "requires": true,
-  "packages": {}
-}
```
Comment (dblock) [inline package-lock.json]: Should we be running an `npm install` to lock dependencies? Or exclude this file in .gitignore. Either way probably a separate PR.

**Review by dblock — CHANGES_REQUESTED** [review]
Thanks! Can you please sign DCO, `git commit -s --amend`.

**Comment by djmadeira** [comment]
@dblock done

**Comment by djmadeira** [comment]
@dblock looks like it's passing now

**Comment by dblock** [comment]
@nhtruong or @Xtansia take a look at the spec?

**Comment by nhtruong** [comment]
This is great, Thanks @djmadeira
Regarding the `x-operation-group`, instead of `rollups.get_rollup`, `rollups.delete_rollup`... we should use `rollups.get`, `rollups.delete` instead to avoid repeating the world rollup as all of these actions are in the `rollups` namespace. We have a similar naming convention in the `indices` namespace with `indices.create`, `indices.delete` and so on.

**Comment by djmadeira** [comment]
@nhtruong addressed feedback

**Comment by nhtruong** [comment]
@djmadeira looks like some `x-operation-group` haven't not been updated. And there are 2 `transforms.get@200` entries and they caused some workflows to fail.

**Comment by djmadeira** [comment]
@nhtruong looks like I got some wires crossed between the two PRs. fixed now.

**Review by nhtruong — APPROVED** [review]

**Pull request merged**